doc/Support/FAQ.md

```diff
@@ -268,7 +262,7 @@ There is a known issue for net-snmp, which causes it to report incorrect disk si
 
 #### <a name="faq8"> How do I add support for a new OS?</a>
 
-Please see [Supporting a new OS](../Developing/Support-New-OS.md)
+Please see [Supporting a new OS](/#Support-for-a-New-OS)
```

Comment:
Links should stay pointing to markdown files, they should get converted correctly whilst ensuring if we change the theme again and a # isn't needed then it doesn't break.